spec/controllers/oauth/applications_controller_spec.rb

```diff
@@ -158,17 +161,11 @@ module Oauth
     end
 
     it "should not let a user get the list of his applications" do
-      controller.sign_in! user
+      controller.sign_in! not_oauth_admin_user
       get(:index)
       expect(response).to have_http_status :forbidden
     end
 
-    it "should not let a user get his own application" do
-      controller.sign_in! user
-      get(:show, params: { id: untrusted_application_user.id })
-      expect(response).to have_http_status :forbidden
-    end
```

Comment:
Put this spec back, change it to succeeding and remove "not" from title?  (because a user can get his app)

Reply:
Done